### PR TITLE
openstack-ardana: merge tempest and QA tests jobs

### PR DESCRIPTION
**Pull request by flaviodsr**
The current implementation does not allow runnig QA tests if any tempest
test (or the tempest stage) failed. To overcome that, the tempest and QA
test jobs have been merged into one job, which makes possible to select
multiple tempest and QA tests and they will be executed independently if the
previous has failed or not.

### CODE REVIEW

File: jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile [hunk 1]
```diff
@@ -84,16 +84,20 @@ pipeline {
     stage('Run tests') {
```
Comment (stefannica) [inline jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile]: Could you also get rid of this old thing, please ? It's just cluttering up the job output.

File: jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile [hunk 2]
```diff
@@ -299,15 +299,19 @@ pipeline {
     stage('Run tests') {
```
Comment (stefannica) [inline jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile]: This too

File: jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile (hunk 1 above)
Comment (flaviodsr) [reply jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile]: Don't we want to keep it for running the `post-deploy-checks` test? (when adding it back)

Comment (stefannica) [reply jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile]: I don't think we'll use the same approach. We'll probably run those in sequence as well.

Comment (flaviodsr) [reply jenkins/ci.suse.de/pipelines/openstack-ardana-update-and-test.Jenkinsfile]: done

File: jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile (hunk 2 above)
Comment (flaviodsr) [reply jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile]: done

**Pull request merged**